### ISSUE TITLE
Cross Entropy "code" example seems like the arguments are reversed

### ISSUE THREAD

**Issue by aaronlelevier** [bfortuner/ml-cheatsheet#28]
In the Loss-Functions > Cross Entropy "code" example the arguments seem reversed.

Location: http://ml-cheatsheet.readthedocs.io/en/latest/loss_functions.html#cross-entropy

It currently reads:

```python
import numpy as np

def CrossEntropy(yHat, y):
    if yHat == 1:
        return -np.log(y)
    else:
        return -np.log(1 - y)
    
print 'true: ', CrossEntropy(.1, 1)
print 'false:', CrossEntropy(.8, 1)

# true:  inf
# false: inf
```

It should read?

```python
def CrossEntropy2(yHat, y):
    if y == 1:
        return -np.log(yHat)
    else:
        return -np.log(1 - yHat)
    
print 'true: ', CrossEntropy2(.1, 1)
print 'false:', CrossEntropy2(.8, 1)

# true:  2.3025850929940455
# false: 0.2231435513142097
```

**Comment by bfortuner** [comment]
Yes you're right! I'll make the fix. Thank!
